charts/vela-minimal/templates/defwithtemplate/config-helm-repository.yaml

```diff
@@ -0,0 +1,59 @@
+# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
+# Definition source cue file: vela-templates/definitions/internal/config-helm-repository.cue
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  annotations:
+    custom.definition.oam.dev/alias.config.oam.dev: Helm Repository
+    definition.oam.dev/description: Config information to authenticate helm chart repository
+  labels:
+    custom.definition.oam.dev/catalog.config.oam.dev: velacore-config
+    custom.definition.oam.dev/multi-cluster.config.oam.dev: "true"
+    custom.definition.oam.dev/type.config.oam.dev: helm-repository
+  name: config-helm-repository
+  namespace: {{ include "systemDefinitionNamespace" . }}
+spec:
+  schematic:
+    cue:
+      template: |
+        output: {
+        	apiVersion: "v1"
+        	kind:       "Secret"
+        	metadata: {
+        		name:      parameter.name
+        		namespace: context.namespace
+        		labels: {
+        			"config.oam.dev/catalog":       "velacore-config"
+        			"config.oam.dev/type":          "helm-repository"
+        			"config.oam.dev/multi-cluster": "true"
+        			"config.oam.dev/identifier":    parameter.registry
+        		}
+        	}
+        	type: "Opaque"
+
+        	if parameter.https != _|_ {
+        		stringData: parameter.https
+        	}
+        	if parameter.ssh != _|_ {
+        		stringData: parameter.ssh
+        	}
+        }
+        parameter: {
```

Comment:
```
repoType: oss | https | ssh
oss?: {
        bucket:   string
        endpoint: string
}
https?: {
      url:      string
      username: string
      password: string
}
// +usage=https://fluxcd.io/legacy/helm-operator/helmrelease-guide/chart-sources/#ssh
ssh?: {
    url:      string
    identity: string
}
```

Reply:
+1 for this

Reply:
Removed Helm Chart repository definition.